### PR TITLE
Add bump version

### PR DESCRIPTION
**Pull request by upupming**
1. Added: https://github.com/PicGo/bump-version
2. Changed git `pre-commit` to husky `pre-commit`
3. Finished setup of azure pipeline, you can now see auto build of the PR in the below!

### CODE REVIEW

**Comment by Molunerfinn** [comment]
Squash then change the commit title to fit `picgo convention`

**Comment by upupming** [comment]
@Spades-S I have reverted to the details/summary style. Does it look good to you now?

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Molunerfinn & Spades
+Copyright (c) 2018-2019 Molunerfinn & Spades
```
Comment (Spades-S) [inline LICENSE]: I think it should be `Copyright (c) 2018-2019 Molunerfinn & Spades & upupMing`

**Comment by Spades-S** [comment]
> @Spades-S I have reverted to the details/summary style. Does it look good to you now?

@upupming  Yeah, LGTM!

File: LICENSE (hunk 1 above)
Comment (upupming) [reply LICENSE]: It will be a great honor and encouragement, thanks and I'll add it!

**Pull request merged**